library/CMakeLists.txt

```diff
@@ -112,6 +112,39 @@ set(src_tls
     ssl_tls13_keys.c
 )
 
+find_package(Perl REQUIRED)
```

Comment:
(Response to Note 2) This is a regression: before, it was possible to build the library without perl, provided that the generated files were present (which we intend to do in releases). Now it isn't: cmake will complain if Perl is not present.

I don't think this regression is desirable. Why is Perl `REQUIRED`? Why not simply let the build error out if perl turns out to be needed but is unavailable?

Reply:
I agree, that's something I left out in this initial version with the intention of coming back to it. I agree that just removing `REQUIRED` should be enough (I copied this line from an example without too much though because I was already absorbed with other issues.)

Reply:
I think that's no longer a problem now that we have the `DEV_MODE` option: Perl is only required when that option is On. Since it will be Off in released tarballs, there will be no regression with released versions.

When development mode is active, I think it's better to get the error early rather than when trying to generate whatever needs Perl to be generated, but if you disagree I can still remove the `REQUIRED` here.